ocs_sample_library_preview/Users.py

```diff
@@ -0,0 +1,224 @@
+import json
+
+from .BaseClient import BaseClient
+from .Security import User
+from .Security import UserInvitation
+
+
+class Users(object):
```

Comment:
should there be a `GetUserById` function?

Reply:
A use case I'm envisioning for this could be that you have the 'owner' of an object, and it says that it's a user with a certain id. You want to get the information for that user, but that's not enough info to build up a full user object to run the `GetOrCreateUser` function. Otherwise you'd have to get the full list of users, but then look for the Id

Reply:
I guess same comment about `GetInvitationById` although to be honest I'm not sure what you'd need that one for. Might be good just for consistency sake

Reply:
I added GetUserById and GetInvitationById for consistency